packages/gatsby/src/utils/webpack-utils.ts

```diff
@@ -675,56 +675,50 @@ export const createWebpackUtils = (
-                  // removeViewBox: true,
-                  removeUselessDefs: true,
-                  addAttributesToSVGElement: true,
-                  addClassesToSVGElement: true,
-                  cleanupAttrs: true,
-                  cleanupEnableBackground: true,
-                  cleanupIDs: true,
-                  cleanupListOfValues: true,
-                  cleanupNumericValues: true,
-                  collapseGroups: true,
-                  convertColors: true,
-                  convertPathData: true,
-                  convertStyleToAttrs: true,
-                  convertTransform: true,
-                  inlineStyles: true,
-                  mergePaths: true,
-                  minifyStyles: true,
-                  moveElemsAttrsToGroup: true,
-                  moveGroupAttrsToElems: true,
-                  prefixIds: true,
-                  removeAttributesBySelector: true,
-                  removeAttrs: true,
-                  removeComments: true,
-                  removeDesc: true,
-                  removeDimensions: true,
-                  removeDoctype: true,
-                  removeEditorsNSData: true,
-                  removeElementsByAttr: true,
-                  removeEmptyAttrs: true,
-                  removeEmptyContainers: true,
-                  removeEmptyText: true,
-                  removeHiddenElems: true,
-                  removeMetadata: true,
-                  removeNonInheritableGroupAttrs: true,
-                  removeOffCanvasPaths: true,
-                  removeRasterImages: true,
-                  removeScriptElement: true,
-                  removeStyleElement: true,
-                  removeTitle: true,
-                  removeUnknownsAndDefaults: true,
-                  removeUnusedNS: true,
-                  removeUselessStrokeAndFill: true,
-                  removeXMLNS: true,
-                  removeXMLProcInst: true,
-                  reusePaths: true,
-                  sortAttrs: true,
-                },
+                // potentially destructive plugins removed - see https://github.com/gatsbyjs/gatsby/issues/15629
+                // use correct config format and remove plugins requiring specific params - see https://github.com/gatsbyjs/gatsby/issues/31619
+                `removeUselessDefs`,
+                `cleanupAttrs`,
+                `cleanupEnableBackground`,
+                `cleanupIDs`,
+                `cleanupListOfValues`,
+                `cleanupNumericValues`,
+                `collapseGroups`,
+                `convertColors`,
+                `convertPathData`,
+                `convertStyleToAttrs`,
+                `convertTransform`,
+                `inlineStyles`,
+                `mergePaths`,
+                `minifyStyles`,
+                `moveElemsAttrsToGroup`,
+                `moveGroupAttrsToElems`,
+                `prefixIds`,
+                `removeAttributesBySelector`,
```

Comment:
This is plugin that seems to require additional configuration and which causes `TypeError: Cannot read property
'length' of undefined` error as no selector is defined it seems (actual crash is later down in stack, but here just showing it get some selector which is empty object and cause problems later).
 
![Screenshot 2021-05-31 at 14 44 20](https://user-images.githubusercontent.com/419821/120195156-b667de80-c21e-11eb-840e-e126dd1e6280.png)

Given that this require configuration - it might also need removing or figuring out default config and using object notation with proper params set?